packages/orbit-components/src/Separator/Separator.stories.tsx

```diff
@@ -77,6 +77,11 @@ export const Playground: Story = {
         type: "select",
       },
     },
+    label: {
```

Comment:
Value for label should be added in `args` (now there is 'Set object' button in SB):
```
args: {
    ...
    label: "Your label",
  },
```

and this could be removed as it's "text" type and it's not necessary ("text" is default type in SB).

Reply:
I don't see the Set object you refer to 🤔

Reply:
We were used to avoiding this button when we were doing SB tasks with domi, and we went by an empty string in `args` object.

I think it's not bad to have there `Set String` button, it was just our convention to avoid it.

Reply:
Alright, I'll have empty string